internal/versions/kube_api_versions_test.go

```diff
@@ -447,31 +428,9 @@ var _ = Describe("ValidateReleaseImageForRHCOS", func() {
 	})
 })
 
-var _ = Describe("GetDefaultReleaseImage", func() {
-	It("Default release image exists", func() {
-		h, err := NewHandler(common.GetTestLog(), nil, defaultReleaseImages, nil, "", nil)
-		Expect(err).ShouldNot(HaveOccurred())
-
-		releaseImage, err := h.GetDefaultReleaseImage(common.TestDefaultConfig.CPUArchitecture)
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(releaseImage.Default).Should(Equal(true))
-		Expect(*releaseImage.OpenshiftVersion).Should(Equal("4.9"))
-		Expect(*releaseImage.CPUArchitecture).Should(Equal(common.TestDefaultConfig.CPUArchitecture))
-	})
-
-	It("Missing default release image", func() {
-		h, err := NewHandler(common.GetTestLog(), nil, models.ReleaseImages{}, nil, "", nil)
-		Expect(err).ShouldNot(HaveOccurred())
-
-		_, err = h.GetDefaultReleaseImage(common.TestDefaultConfig.CPUArchitecture)
-		Expect(err).Should(HaveOccurred())
-		Expect(err.Error()).Should(Equal("Default release image is not available"))
-	})
-})
-
 var _ = Describe("GetMustGatherImages", func() {
```

Comment:
I think we can be verbose in this one and keep it in separate tests as it uses `GetReleaseImage` for specific instance

Reply:
But the content is exactly the same.
I don't see value in having to change identical tests in two places when we make one code change unless there is significant differences between how the function is used, which is not the case here.

Reply:
I tried to convert it, in my opinion it makes less sense in common as the test itself doesn't have much context but calling the function with different parameters, therefore it turns up as a bunch of restAPI calls (with restAPI.GetReleaseImage) and then a bunch of kubeAPI calls, same as if they were in separate files. In my opinion, it would make more sense to keep them as they are currently, but if you disagree I can merge the tests in common_test.go

Reply:
From meeting: You can "mock" the `GetReleaseImage` by passing in a function defined just for the test to return some test release image. Don't worry about the different real implementations of `GetReleaseImage`